moduleframework/helpers/container_helper.py

```diff
@@ -128,35 +165,18 @@ def start(self, args="-it -d", command="/bin/bash"):
         :return: None
         """
         if not self.status():
-            if 'start' in self.info and self.info['start']:
+            if self.info.get('start'):
                 self.docker_id = self.runHost(
-                    "%s -d %s" %
-                    (self.info['start'], self.jmeno), shell=True, ignore_bg_processes=True,
+                    "%s -d %s %s" %
+                    (self.info['start'], self.docker_static_name, self.jmeno), shell=True, ignore_bg_processes=True,
                     verbose=is_not_silent()).stdout
             else:
                 self.docker_id = self.runHost(
-                    "docker run %s %s %s" %
-                    (args, self.jmeno, command), shell=True, ignore_bg_processes=True, verbose=is_not_silent()).stdout
+                    "docker run %s %s %s %s" %
+                    (args, self.docker_static_name, self.jmeno, command), shell=True, ignore_bg_processes=True, verbose=is_not_silent()).stdout
             self.docker_id = self.docker_id.strip()
-            if self.getPackageList():
-                a = self.run(
-                    "%s install %s" %
-                    (trans_dict["HOSTPACKAGER"], " ".join(
-                        self.getPackageList())),
-                    ignore_status=True, verbose=False)
-                b = self.run(
-                    "%s install %s" %
-                    (trans_dict["GUESTPACKAGER"], " ".join(
-                        self.getPackageList())),
-                    ignore_status=True, verbose=False)
-                if a.exit_status == 0:
-                    print_info("Packages installed via {HOSTPACKAGER}", a.stdout)
-                elif b.exit_status == 0:
-                    print_info("Packages installed via {GUESTPACKAGER}", b.stdout)
-                else:
-                    print_info(
-                        "Nothing installed (nor via {HOSTPACKAGER} nor {GUESTPACKAGER}), but package list is not empty",
-                        self.getPackageList())
+            # It installs packages in container is removed by default, in future maybe reconciled.
```

Comment:
I don't follow this line at all. Please explain, so I can edit it accordingly.

Reply:
In past. I had to ensure installation of packages also for docker, because cotainers could be also empty. So I had this code inside this function. But after discussion, that actually, nobody uses this idea of empty containers, and containers have got everytime installed packages. This package installation causes performance regression.  So that I've moved this code to own method, what is disabled for now. It may happen that this idea will appear again, so then it will be easier just enable it, than write it again or look for it in git history.